util/desktop/paths_linux.go

```diff
@@ -0,0 +1,25 @@
+package desktop
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+)
+
+const (
+	socketName    = "docker-desktop-build.sock"
+	socketPath    = ".docker/desktop"
+	wslSocketPath = "/mnt/wsl/docker-desktop/shared-sockets/host-services"
+)
+
+func BuildServerAddr() (string, error) {
+	if os.Getenv("WSL_DISTRO_NAME") != "" {
+		return "unix://" + filepath.Join(wslSocketPath, socketName), nil
+	}
```

Comment:
Pushed this change for testing WSL support but doesn't work. I'm taking a look if this can be solved but in the meantime we could disable support for `import` command on WSL and explain that it should be invoked on Windows host instead while this is being investigated.